christfellowshipchurch/apollos-app
# Updated the Push Notifications settings button

### About
This PR updates the push notifications settings button. Due to the ApollosProject not being able to properly check if someone has enabled push notifications before we have updated our settings button UI to just show a "Manage" button that will point the user to settings to manage their push notifications.

### Test Instructions
* run app locally
* go to  Connect -> Edit User -> Push Notifications
* Check UI and button to make sure it properly goes to Settings.

### Screenshots
![image](https://user-images.githubusercontent.com/46049974/141517228-cd200a31-5204-4f49-8f5b-c15f4056ffdb.png)

### Closes Tickets
[CFDP-1759]


[CFDP-1759]: https://christfellowshipchurch.atlassian.net/browse/CFDP-1759?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
